test/index.js

```diff
@@ -267,10 +267,20 @@ describe('electronjs.org', () => {
     })
   })
 
-  test('/languages', async () => {
-    const $ = await get('/languages')
-    $('h1').text().should.eq('Languages')
-    $('body').text().should.include('global developer community')
+  describe('languages', () => {
+    test('/languages', async () => {
+      const $ = await get('/languages')
+      $('h1').text().should.eq('Languages')
+      $('body').text().should.include('global developer community')
+    })
+
+    test('language query param for one-off viewing in other languages', async () => {
+      let $ = await get('/docs/api/browser-window')
+      $('body').text().should.not.include('fenêtres')
+
+      $ = await get('/docs/api/browser-window?language=fr-FR')
+      $('body').text().should.include('fenêtres')
```

Comment:
Should we swap the order of these two tests, so we know the `?language=fr-FR` param isn't affecting the language stored in the cookies?

Reply:
Good call. Done.